031_sumOfdigitsOfNumber.js

```diff
@@ -0,0 +1,12 @@
+// 31. Sum of digits of a number
+const sumOfDigits = (number) => {
```

Comment:
logic issue : Number should be read in as an Integer. It should not be converted to a string or read in as string